cli/src/commands/generate/components/node.ts

```diff
@@ -115,7 +115,7 @@ export function instantiateNodeInterfaces(detail: any, schemaDirectory: SchemaDi
     const logger = initLogger(debug);
     const interfaces = [];
     if (!('prefixItems' in detail)) {
-        logger.error('No items in interfaces block.');
+        logger.debug('Instantiating a node with no interfaces defined.');
```

Comment:
I don't think we want to log on this, in fact the schema needs to be updated as interfaces are only valid on certain types of node.